### PR TITLE
Reviewing docs for errors and mistakes uptill concepts section

### PR DESCRIPTION
**Pull request by dino-saurabh**
1.concept.md
2.concept_attribute_type.md
3.concept_data_type.md
4.concept_map_type.md
5.concept_reference_term_type.md
6.concept_source.md
7.concept_class.md
8.encounter.md
9.encounter_role.md
10.encounter_type.md

in addition fixed few clashes and UI errors as well in this section 
1. concept.md
2. concept_attribute_type.md

### CODE REVIEW

**Comment by dino-saurabh** [comment]
Before

createPrev

After

after

before 

updatePrev

after

afterupdate

**Comment by dino-saurabh** [comment]
before for concept attribute type.md

before

after 

after

**Comment by dino-saurabh** [comment]
@Akayeshmantha please review .

File: source/includes/Concepts/concept.md [hunk 1]
```diff
@@ -5,17 +5,15 @@
...
+("blood type") and each response ("A," "B," "AB" and "O") is also a concept.  For this one question, a total of 5 concepts are required.
 
 * What about a question where the answer is not a discrete answer?  If the question is "What is the name of your first pet?",
 the answer would be expressed in a text box.  It would not be possible to provide a complete list of every possible name for
-your pet.  In this example, there would be one concept -- "name of first pet".
+your pet.  In this example, there would be one concept -- "name of first pet."
 
-* The bottom line is, if you need a medical word within your electronic records system, it needs to be defined within the 
-concept dictionary.  More detail about all the possible concepts in a later section.
+* The bottom line is that if you need a medical word within your electronic records system, it needs to be defined within the concept dictionary, more details about all the possible concepts will be given in a later section.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md]:
```suggestion
* The bottom-line is that if you need a medical word within your electronic records system, it needs to be defined within the concept dictionary, more details about all the possible concepts will be given in a later section.
```

File: source/includes/Concepts/concept.md [hunk 2]
```diff
@@ -83,7 +79,7 @@ referenced source.
...
-    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification details how a concept will be represented (i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes for use.
+    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification elaborates how a concept will be represented (i.e., as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md]:
```suggestion
    *class* | `String` | The concept's class provides a useful way to narrow down the scope of the information that the concept is intended to capture.  In this way, the class is helpful for data extraction.  This classification elaborates how a concept will be represented (i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when defining concepts, but implementation sites may also create additional custom concept classes.
```

File: source/includes/Concepts/concept.md [hunk 3]
```diff
@@ -93,16 +89,15 @@ GET /concept?
...
+    Retrieve a concept by its UUID. Returns a `404 Not Found` status if concepts not exist. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
     
 ```console
 GET /concept/:target_concept_uuid
 ```
    
 ## Create a concept
 
-* To Create an concept you need to specify below attributes in the request body.If you are not logged in to perform this action,
+* To Create a concept, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:100]:
```suggestion
* To create a concept, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

File: source/includes/Concepts/concept.md [hunk 4]
```diff
@@ -139,7 +134,7 @@ POST /concept
 ## Update a concept
 
 *  Update a target concept with given UUID, this method only modifies properties in the request. Returns a `404 Not Found` 
-status if concept not exists. If user not logged in to perform this action, a `401 Unauthorized` status returned.
+status if concept not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md]:
```suggestion
status if concept not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status is returned.
```

File: source/includes/Concepts/concept.md [hunk 5]
```diff
@@ -191,34 +186,32 @@ DELETE /concept/:target_concept_uuid?purge=true
...
+     `404 Not Found` status if concept mapping not exists. If you are not logged in to perform this action, a `401 Unauthorized` status returned.
      
 ```console
 GET /concept/:target_concept_uuid/mapping/:target_concept_mapping_uuid
 ```
 ## Create a concept mapping with properties
 
-* To Create a concept mapping sub resource for a specific concept resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create a concept mapping subresource for a specific concept resource, you need to specify below attributes in the request body.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:206]:
```suggestion
* To create a concept mapping sub-resource for a specific concept resource, you need to specify below attributes in the request body.
```

File: source/includes/Concepts/concept.md [hunk 6]
```diff
@@ -231,16 +224,15 @@ POST concept/:target_concept_uuid/mapping
  
 ## Update a concept mapping
 
-* Updates an concept mapping sub resource value with given uuid, this method will only modify value of the sub resource. Returns 
-a `404 Not Found` status if concept mapping not exists. If user not logged in to perform this action, a `401 Unauthorized` status
+* Updates a concept mapping subresource value with given UUID, this method will only modify the value of the subresource. Returns a `404 Not Found` status if concept mapping not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:227]:
```suggestion
* Updates a concept mapping sub-resource value with given UUID, this method will only modify the value of the sub-resource. Returns a `404 Not Found` status if concept mapping not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status
```

File: source/includes/Concepts/concept.md [hunk 7]
```diff
@@ -251,14 +243,14 @@ POST concept/:target_concept_uuid/mapping
 ```
 ## Delete a concept mapping
 
-* Delete or Voided a target concept mapping sub resource by its UUID. Returns a `404 Not Found` status if concept mapping not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or Voided a target concept mapping subresource by its UUID. Returns a `404 Not Found` status if concept mapping not exists.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:246]:
```suggestion
* Delete or Voided a target concept mapping sub-resource by its UUID. Returns a `404 Not Found` status if concept mapping not exists. 
```

File: source/includes/Concepts/concept.md [hunk 8]
```diff
@@ -267,35 +259,33 @@ DELETE concept/:target_concept_uuid/mapping/:target_concept_mapping_uuid
 
 * ### List all concept names for a concept.
 
-    Retrieve all **concept name** sub resources of a **concept** resource by target_concept_uuid. Returns a 
-    `404 Not Found` status if concept name not exists. If user not logged in to perform this action, a `401 Unauthorized` status
-    returned.
+    Retrieve all **concept name** subresources of a **concept** resource by target_concept_uuid. Returns a
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:262]:
```suggestion
    Retrieve all **concept name** sub-resources of a **concept** resource by target_concept_uuid. Returns a 
```

File: source/includes/Concepts/concept.md [hunk 9]
```diff
@@ -267,35 +259,33 @@ DELETE concept/:target_concept_uuid/mapping/:target_concept_mapping_uuid
 
 * ### List all concept names for a concept.
 
-    Retrieve all **concept name** sub resources of a **concept** resource by target_concept_uuid. Returns a 
-    `404 Not Found` status if concept name not exists. If user not logged in to perform this action, a `401 Unauthorized` status
-    returned.
+    Retrieve all **concept name** subresources of a **concept** resource by target_concept_uuid. Returns a 
+    `404 Not Found` status if concept name not exists. If the user isnot logged in to perform this action, a `401 Unauthorized` status returned.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:263]:
```suggestion
    `404 Not Found` status if concept name not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
```

File: source/includes/Concepts/concept.md [hunk 10]
```diff
@@ -267,35 +259,33 @@ DELETE concept/:target_concept_uuid/mapping/:target_concept_mapping_uuid
...
+     `404 Not Found` status if concept name not exists. If you are not logged in to perform this action, a `401 Unauthorized` status returned.
      
 ```console
 GET /concept/:target_concept_uuid/name/:target_concept_name_uuid
 ```
 ## Create a concept name with properties
 
-* To Create a concept name sub resource for a specific concept resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create a concept name subresource for a specific concept resource, you need to specify below attributes in the request body.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:279]:
```suggestion
* To Create a concept name sub-resource for a specific concept resource, you need to specify below attributes in the request body.
```

File: source/includes/Concepts/concept.md [hunk 11]
```diff
@@ -311,8 +301,7 @@ POST concept/:target_concept_uuid/name
  
 ## Update concept name
 
-* Updates an concept name sub resource value with given uuid, this method will only modify value of the sub resource. Returns 
-a `404 Not Found` status if concept name not exists. If user not logged in to perform this action, a `401 Unauthorized` status
+* Updates a concept name subresource value with given UUID, this method will only modify value of the subresource. Returns a `404 Not Found` status if concept name not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:304]:
```suggestion
* Updates a concept name sub-resource value with given UUID, this method will only modify value of the sub-resource. Returns a `404 Not Found` status if concept name not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status
```

File: source/includes/Concepts/concept.md [hunk 12]
```diff
@@ -335,14 +324,14 @@ POST concept/:target_concept_uuid/name
 ```
 ## Delete concept name
 
-* Delete or retire a target concept name sub resource by its UUID. Returns a `404 Not Found` status if concept name not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or retire a target concept name subresource by its UUID. Returns a `404 Not Found` status if concept name not exists.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:327]:
```suggestion
* Delete or retire a target concept name sub-resource by its UUID. Returns a `404 Not Found` status if concept name not exists. 
```

File: source/includes/Concepts/concept.md [hunk 13]
```diff
@@ -352,27 +341,25 @@ DELETE concept/:target_concept_uuid/name/:target_concept_name_uuid
...
 GET /concept/:target_concept_uuid/attribute 
 ```
 
-* ### List concept attribute by it's UUID and parent concept UUID.
+* ### List concept attribute by its UUID and parent concept UUID.
     
-     Retrieve an **concept attribute** sub resources of a **concept** resource. Returns a 
-     `404 Not Found` status if concept attribute not exists. If you are not logged in to perform this action, a `401 Unauthorized` status
-     returned.
+     Retrieve a **concept attribute** subresources of a **concept** resource. Returns a
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:353]:
```suggestion
     Retrieve a **concept attribute** sub-resources of a **concept** resource. Returns a 
```

File: source/includes/Concepts/concept.md [hunk 14]
```diff
@@ -352,27 +341,25 @@ DELETE concept/:target_concept_uuid/name/:target_concept_name_uuid
...
+     `404 Not Found` status if a concept attribute not exists. If you are not logged in to perform this action, a `401 Unauthorized` status returned.
      
 ```console
 GET /concept/:target_concept_uuid/attribute/:target_concept_attribute_uuid
 ```
 ## Create a concept attribute with properties
 
-* To Create a concept attribute sub resource for a specific concept resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create a concept attribute subresource for a specific concept resource, you need to specify below attributes in the request body.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:361]:
```suggestion
* To create a concept attribute sub-resource for a specific concept resource, you need to specify below attributes in the request body.
```

File: source/includes/Concepts/concept.md [hunk 15]
```diff
@@ -412,14 +398,14 @@ POST concept/:target_concept_uuid/attribute
 ```
 ## Delete concept attribute
 
-* Delete or retire a target concept attribute sub resource by its UUID. Returns a `404 Not Found` status if concept attribute not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or retire a target concept attribute subresource by its UUID. Returns a `404 Not Found` status if concept attribute not exists.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:401]:
```suggestion
* Delete or retire a target concept attribute sub-resource by its UUID. Returns a `404 Not Found` status if concept attribute not exists. 
```

File: source/includes/Concepts/concept.md [hunk 16]
```diff
@@ -429,27 +415,25 @@ DELETE concept/:target_concept_uuid/attribute/:target_concept_attribute_uuid
...
+     `404 Not Found` status if concept description not exists. If you are not logged in to perform this action, a `401 Unauthorized` status returned.
      
 ```console
 GET /concept/:target_concept_uuid/description/:target_concept_description_uuid
 ```
 ## Create a concept description with properties
 
-* To Create a concept description sub resource for a specific concept resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create a concept description subresource for a specific concept resource you need to specify below attributes in the request body.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:435]:
```suggestion
* To create a concept description sub-resource for a specific concept resource you need to specify below attributes in the request body.
```

File: source/includes/Concepts/concept.md [hunk 17]
```diff
@@ -483,20 +466,20 @@ returned.
...
-  "description": "Pregnancy terminated by a spontaneous abortion.",
+  "description": "Pregnancy terminated by spontaneous abortion.",
   "locale": "en"
 }
 ```
 ## Delete concept description
 
-* Delete or retire a target concept description sub resource by its UUID. Returns a `404 Not Found` status if concept description not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or retire a target concept description subresource by its UUID. Returns a `404 Not Found` status if concept description not exists.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept.md:475]:
```suggestion
* Delete or retire a target concept description sub-resource by its UUID. Returns a `404 Not Found` status if concept description not exists. 
```

File: source/includes/Concepts/concept_attribute_type.md [hunk 18]
```diff
@@ -31,32 +31,30 @@ GET /conceptattributetype?q=time
...
 
 ```console
 GET /conceptattributetype/:target_concept_attribute_type_uuid
 ```
 
 ## Create a concept attribute type
 
-* To Create a concept attribute type you need to specify below attributes in 
-  the request body.If you are not logged in to perform this action,
+* To Create a concept attribute type, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_attribute_type.md:42]:
```suggestion
* To create a concept attribute type, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

File: source/includes/Concepts/concept_map_type.md [hunk 19]
```diff
@@ -2,14 +2,14 @@
...
-* Concept Mappings are added to facilitate managing concept dictionaries and point to other concepts which have the same meaning. 
+* Concept Mappings are added to facilitate managing concept dictionaries and point to other concepts that have the same meaning. 
 Mappings are useful when you need to receive or send information to external systems, letting you define how your system's 
 concepts relate to external concepts such as standardized medical vocabularies (e.g., ICD, LOINC, SNOMED). 
   
 * For example, add a mapping to a concept in the MCL dictionary. You can save the concept now and create some answers.
 
 * Repeat the steps and create the concepts PLANNING PREGNANCY and CURRENTLY PREGNANT of Class Finding and Datatype Boolean. 
-The last possible answer will be OTHER of Class Misc and Datatype N/A. After creating three new concepts, you can edit 
+The last possible answer will be the OTHER of Class Misc and Datatype N/A. After creating three new concepts, you can edit
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_map_type.md:12]:
```suggestion
The last possible answer will be OTHER of Class Misc and Datatype N/A. After creating three new concepts, you can edit 
```

File: source/includes/Concepts/concept_map_type.md [hunk 20]
```diff
@@ -89,8 +89,7 @@ status if concept map not exists. If user not logged in to perform this action,
 
 ### Delete a concept map type
 
-* Delete or Retire a target concept map type by its UUID. Returns a `404 Not Found` status if concept map not exists.If user not logged 
-  in to perform this action, a `401 Unauthorized` status returned.
+* Delete or Retire a target concept map type by its UUID. Returns a `404 Not Found` status if concept map not exists. If user not logged in to perform this action, a `401 Unauthorized` status returned.
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_map_type.md:92]:
```suggestion
* Delete or retire a target concept map type by its UUID. Returns a `404 Not Found` status if concept map not exists. If user not logged in to perform this action, a `401 Unauthorized` status returned.
```

File: source/includes/Concepts/concept_map_type.md [hunk 21]
```diff
@@ -48,7 +48,7 @@ ANTENATAL VISIT REASON and add them as answers.
 
 ### Create a concept map type
 
-* To Create a concept map type you need to specify below attributes in the request body. If you are not logged in to perform this action,
+* To Create a concept map type, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_map_type.md:51]:
```suggestion
* To create a concept map type, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

File: source/includes/Concepts/concept_reference_term_type.md [hunk 22]
```diff
@@ -33,26 +30,25 @@
...
+    Retrieve a concept reference term by its UUID. Returns a `404 Not Found` status if concept reference term not exists. If the user is not logged in to perform this action, a `401 Unauthorized` status returned.
 
     ```console
     GET /conceptreferenceterm/:target_concept_reference_term_type_uuid
     ```
 
 ### Create a concept reference term
 
-* To Create an concept reference term you need to specify below attributes in the request body.If you are not logged in to perform this action,
+* To Create an concept reference term, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_reference_term_type.md:41]:
```suggestion
* To create an concept reference term, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

File: source/includes/Concepts/concept_reference_term_type.md [hunk 23]
```diff
@@ -33,26 +30,25 @@
...
     *code* | `String` | Represents a name from a standard medical code (required)
-    *conceptSource* | `target_concept_source_UUID` | A concept can have any number of mappings to any number of other vocabularies. Other vocabularies are called "concept sources" in OpenMRS (ie. LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc), but the concept source can also be a custom (ie. org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). Every concept can define a string for its mapping in any "concept source" defined in the database (required)
+    *conceptSource* | `target_concept_source_UUID` | A concept can have any number of mappings to any number of other vocabularies. Other vocabularies are called "concept sources" in OpenMRS (i.e., LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc.), but the concept source can also be a custom (i.e., org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). Every concept can define a string for its mapping in any "concept source" defined in the database (required)
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_reference_term_type.md:51]:
```suggestion
    *conceptSource* | `target_concept_source_UUID` | A concept can have any number of mappings to any number of other vocabularies. Other vocabularies are called "concept sources" in OpenMRS (i.e. LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc.), but the concept source can also be a custom (i.e. org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). Every concept can define a string for its mapping in any "concept source" defined in the database (required)
```

File: source/includes/Concepts/concept_reference_term_type.md [hunk 24]
```diff
@@ -75,7 +71,7 @@ status if concept reference term not exists. If user not logged in to perform th
...
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concept_reference_term_type.md:74]:
```suggestion
    *conceptSource* | `target_concept_source_UUID` | A concept can have any number of mappings to any number of other vocabularies. Other vocabularies are called "concept sources" in OpenMRS (i.e. LOINC, SNOMED, ICD-9, ICD10, RxNORM, etc.), but the concept source can also be a custom (i.e. org.openmrs.module.mdrtb, PIH, AMPATH, MVP, etc.). Every concept can define a string for its mapping in any "concept source" defined in the database (required)
```

File: source/includes/Concepts/concepts_class.md [hunk 25]
```diff
@@ -37,7 +36,7 @@ defining concepts, but implementation sites may also create additional custom co
 
 ### Create a concept class
 
-* To Create a concept class you need to specify below attributes in the request body. If you are not logged in to perform this action,
+* To Create a concept class, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```
Comment (sanjulamadurapperuma) [inline source/includes/Concepts/concepts_class.md:39]:
```suggestion
* To create a concept class, you need to specify below attributes in the request body. If you are not logged in to perform this action,
```

File: source/includes/Encounter/encounter.md [hunk 26]
```diff
@@ -164,9 +162,9 @@ POST /encounter/:target_encounter_uuid
 ```console
   DELETE /encounter/:target_encounter_uuid?purge=true
 ```
-## List encounter provider sub resources
+## List encounter provider subresources
 
-* ### List all encounter provider sub resources for a visit.
+* ### List all encounter provider subresources for a visit.
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:167]:
```suggestion
* ### List all encounter provider sub-resources for a visit.
```

File: source/includes/Encounter/encounter.md [hunk 27]
```diff
@@ -164,9 +162,9 @@ POST /encounter/:target_encounter_uuid
 ```console
   DELETE /encounter/:target_encounter_uuid?purge=true
 ```
-## List encounter provider sub resources
+## List encounter provider subresources
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:165]:
```suggestion
## List encounter provider subresources
```
```suggestion
## List encounter provider sub-resources
```

File: source/includes/Encounter/encounter.md [hunk 28]
```diff
@@ -176,7 +174,7 @@ POST /encounter/:target_encounter_uuid
 GET /encounter/:target_encounter_uuid/encounterprovider 
 ```
 
-* ### List encounter provider sub resources by it's UUID and parent encounter UUID.
+* ### List encounter provider subresources by it's UUID and parent encounter UUID.
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:177]:
```suggestion
* ### List encounter provider sub-resources by it's UUID and parent encounter UUID.
```

File: source/includes/Encounter/encounter.md [hunk 29]
```diff
@@ -187,8 +185,8 @@ GET /encounter/:target_encounter_uuid/encounterprovider/:target_encounter_provid
 ```
 ## Create an encounter provider sub resource with properties
 
-* To Create an attribute sub resource for a specific visit resource you need to specify below attributes in the request body.
-If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* To Create an attribute subresource for a specific visit resource, you need to specify below attributes in the request body.
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:188]:
```suggestion
* To create an attribute sub-resource for a specific visit resource, you need to specify below attributes in the request body.
```

File: source/includes/Encounter/encounter.md [hunk 30]
```diff
@@ -206,10 +204,9 @@ POST encounter/:target_encounter_uuid/encounterprovider
 ```
  
  
-## Update encounter provider sub resource
+## Update encounter provider subresource
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:207]:
```suggestion
## Update encounter provider sub-resource
```

File: source/includes/Encounter/encounter.md [hunk 31]
```diff
@@ -233,16 +230,16 @@ POST encounter/:target_encounter_uuid/encounterprovider/:target_encounter_provid
   "encounterRole": "240b26f9-dd88-4172-823d-4a8bfeb7841f"
 }
 ```
-## Delete encounter provider sub resource
+## Delete encounter provider subresource
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:233]:
```suggestion
## Delete encounter provider sub-resource
```

File: source/includes/Encounter/encounter.md [hunk 32]
```diff
@@ -233,16 +230,16 @@ POST encounter/:target_encounter_uuid/encounterprovider/:target_encounter_provid
...
 
-* Delete or Voided a target encounter provider sub resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
- If user not logged in to perform this action, a `401 Unauthorized` status returned.
+* Delete or Voided a target encounter provider subresource by its UUID. Returns a `404 Not Found` status if attribute not exists.
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter.md:235]:
```suggestion
* Delete or Voided a target encounter provider sub-resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
```

File: source/includes/Encounter/encounter_type.md [hunk 33]
```diff
@@ -45,7 +44,7 @@ GET /encountertype/:target_encounter_type_uuid
    
 ## Create an encounter type
 
-* To Create an encounter type you need to specify below attributes in the request body. If you are not logged in to perform 
+* To Create an encounter type, you need to specify below attributes in the request body. If you are not logged in to perform
```
Comment (sanjulamadurapperuma) [inline source/includes/Encounter/encounter_type.md]:
```suggestion
* To create an encounter type, you need to specify below attributes in the request body. If you are not logged in to perform 
```

**Review by sanjulamadurapperuma — CHANGES_REQUESTED** [review]
Requested some changes

**Comment by dino-saurabh** [comment]
@sanjulamadurapperuma thanks for reviewing my PR I will make the necessary changes 😀

File: source/includes/Encounter/encounter.md (hunk 32 above)
Comment (dino-saurabh) [reply source/includes/Encounter/encounter.md:235]: @sanjulamadurapperuma In the docs in other sections as well subresource is used uniformly all over thats why I have adhered to it here as well.

File: source/includes/Concepts/concept.md (hunk 2 above)
Comment (dino-saurabh) [reply source/includes/Concepts/concept.md]: @sanjulamadurapperuma actually I think its notationally correct to use a comma after i.e. Like the one I have used above If its not I will change it 🙂

File: source/includes/Concepts/concept.md (hunk 3 above)
Comment (dino-saurabh) [reply source/includes/Concepts/concept.md:100]: @sanjulamadurapperuma all the operations were starting from a capital letter so should I change them all like update delete as well ?

File: source/includes/Encounter/encounter.md (hunk 32 above)
Comment (sanjulamadurapperuma) [reply source/includes/Encounter/encounter.md:235]: @LuGO0 It should actually be _sub-resources_ in any context. We have used _sub-resources_ in our [Wiki](https://wiki.openmrs.org/display/docs/Documenting+REST+Resources) as well. Could you point to some other doc sections using _subresources_?

File: source/includes/Concepts/concept.md (hunk 3 above)
Comment (sanjulamadurapperuma) [reply source/includes/Concepts/concept.md:100]: I think it is better to because it actually is a verb, not the operation name. @dkayiwa @Akayeshmantha appreciate your thoughts on this.

File: source/includes/Concepts/concept.md (hunk 2 above)
Comment (sanjulamadurapperuma) [reply source/includes/Concepts/concept.md]: It's actually in American English. They use **i.e,** with the comma instead of the last full-stop/period. If we are okay with following that notation, then I am +1 to have **i.e,** instead of **i.e.,** which seems out-of-place.

File: source/includes/Concepts/concept.md (hunk 5 above)
Comment (LuGO0) [reply source/includes/Concepts/concept.md:206]: Okay @sanjulamadurapperuma I will make the changes 🙂

File: source/includes/Concepts/concept.md (hunk 2 above)
Comment (dino-saurabh) [reply source/includes/Concepts/concept.md]: Okay @sanjulamadurapperuma I will make the changes 🙂

**Review by Akayeshmantha — APPROVED** [review]
@LuGO0  good work I like the suggestions but lets now focus on adding javascript samples.
